README.md

```diff
@@ -764,7 +764,7 @@ This section contains advice about integrating specific build tools with Wireit.
       "command": "tsc --build --pretty",
       "clean": "if-file-deleted",
       "files": ["src/**/*.ts", "tsconfig.json"],
-      "output": ["lib/**", ".tsbuildinfo"]
+      "output": ["lib/**", "*.tsbuildinfo"]
```

Comment:
If we're going by default behavior, this might be better. See https://github.com/google/wireit/issues/505#issuecomment-1306485439
```suggestion
      "output": ["lib/**", "lib/tsconfig.tsbuildinfo"]
```

Reply:
My `tsconfig.tsbuildinfo` file doesn't get output to the output folder. And otherwise it would have been included by the `lib/**` glob, right? I chose to use `*.tsbuildinfo` specifically because the name of the file can be different depending on your `tsconfig.json`. See: https://www.typescriptlang.org/tsconfig#tsBuildInfoFile.

Reply:
@augustjk Did you happen to double check the default behavior here already?

Reply:
Just tested now. Seems like I was mistaken, but so is the doc linked above.
> If `outDir` is set, then the default is `<outDir>/<config name>.tsbuildInfo`

But I observed `tsconfig.tsbuildinfo` being generated next to the config file, **not** inside the `<outDir>`.

I retract the suggestion.